contracts/test/recovery-test.ts

```diff
@@ -327,10 +330,13 @@ describe("Recovery", async function () {
       recoveredWalletNonce++, // await wallet2.Nonce(),
     );
 
+    console.log("here1");
     expect(await vg.walletFromHash(hash1)).to.not.equal(blsWallet.address);
+    console.log("here2");
     expect(await vg.walletFromHash(hashAttacker)).to.not.equal(
```

Comment:
This is the line this test fails on @blakecduncan

Reply:
Fixed by using the attacker's `blsWalletSigner` to create the `attackerHash`, instead of wallet 2s signer.